### PR TITLE
fix: avoid deprecated methods

### PR DESCRIPTION
**Pull request by blizzz**

### CODE REVIEW

File: lib/Listener/RegisterOperationsListener.php [hunk 1]
```diff
@@ -0,0 +1,49 @@
+<?php
+
+declare(strict_types=1);
+/**
+ * @copyright Copyright (c) 2024 Arthur Schiwon <blizzz@arthur-schiwon.de>
```
Comment (ChristophWurst) [inline lib/Listener/RegisterOperationsListener.php:5]: nit: spdx to make Andy happy

**Review by ChristophWurst — APPROVED** [review]
Very nice

This makes the app very light to boot  :balloon::house::balloon:

File: lib/Listener/RegisterOperationsListener.php (hunk 1 above)
Comment (blizzz) [reply lib/Listener/RegisterOperationsListener.php:5]: thought about that, but the remainder of the app is not. consistency? :grin:

Comment (ChristophWurst) [reply lib/Listener/RegisterOperationsListener.php:5]: Fair point. Then it might be better to leave the old style and convert later :+1:

**Review by juliushaertl — APPROVED** [review]
🚀

**Pull request merged**